examples/android/CHIPTool/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.google.chip.chiptool">
 
+    <uses-sdk android:minSdkVersion="28" />
```

Comment:
What specifically requires 28 ? Could it have been 26?

Reply:
I believe there was more than one API dependency triggering this, but the only one I can find in my build failures is the recommendation of using >= 28 when using NsdManager (in NsdManagerServiceBrowser)